pymc3/distributions/continuous.py

```diff
@@ -4226,6 +4226,31 @@ class Interpolated(BoundedContinuous):
     ========  ===========================================
     Support   :math:`x \in [x\_points[0], x\_points[-1]]`
     ========  ===========================================
+    
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pymc3 as pm
+    import warnings
+    warnings.simplefilter(action="ignore", category=FutureWarning)
+    from scipy.stats import gamma
+    a = 1.99
+    mean, var, skew, kurt = gamma.stats(a, moments='mvsk')
+    rv = gamma(a)
+    x = np.linspace(rv.ppf(0.01),rv.ppf(0.99), 1000)
+    points = np.linspace(x[0], x[-1], 50)
+    pdf = rv.pdf(points)
+    interpolated = pm.Interpolated.dist(points, pdf)
+    plt.style.use('arviz-darkgrid')
+    plt.style.use('tableau-colorblind10')
+    #plt.set_cmap('coolwarm')
```

Comment:
```suggestion
    plt.style.use('arviz-darkgrid')
```

Following on the last issue comment, only the import and this line are needed to use `arviz-darkgrid` style, there is no need for inferencedata or any other arviz related thing. We may not be very good at advertising it, but ArviZ has many cool features in addition to inferencedata and functions that take inferencedata as input.